src/core/src/lib/extensions/field-form/field-form.ts

```diff
@@ -8,7 +8,7 @@ import { registerControl } from './utils';
 export class FieldFormExtension implements FormlyExtension {
   onPopulate(field: FormlyFieldConfigCache) {
     // TODO: add an option to skip extension
-    if (field.fieldArray) {
+    if (field.fieldArray || field.type === 'array') {
```

Comment:
use `field.hasOwnProperty('fieldArray')` instead

Reply:
The problem with this is that:
1. This won't support prototypal inheritance if someone wanted to extend form configuration, which is quite natural in JS.
2. This will work improperly when some one generates configuration with `fieldArray: null` or `fieldArray: undefined`.
3. This will be a breaking change because of the above.

Are you absolutely sure? If so, I think this should be documented.

Reply:
right 👍, I've got another way to fix the `json-schema` issue, instead of `null` value I used an empty object when `additionalItems` is empty.